ui/components/institutional/interactive-replacement-token-modal/interactive-replacement-token-modal.js

```diff
@@ -9,11 +9,11 @@ import {
   Box,
   Button,
   Modal,
-  ModalContent,
-  ModalHeader,
   ModalOverlay,
   Text,
 } from '../../component-library';
+import { ModalContent } from '../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../component-library/modal-header/deprecated';
 
 import {
   BlockSize,
```

Comment:
Screencast shows no difference to scrolling but subtle difference to vertical padding on small screen heights

### Before


https://github.com/MetaMask/metamask-extension/assets/8112138/643285bd-195c-4b15-868c-f0d3c90e6367

### After


https://github.com/MetaMask/metamask-extension/assets/8112138/ff43e67f-33c7-4ded-9356-0de849de674c